src/WelcomeBody.css

```diff
@@ -0,0 +1,11 @@
+/** Scaffolding **/
+#welcomeBody {
```

Comment:
For whatever reasons, camel casing is not done in CSS usually. 
Make it all lowercase or use hyphens.

Reply:
Updated.